src/main/java/org/broadinstitute/dsm/model/fieldsettings/FieldSettings.java

```diff
@@ -45,4 +57,11 @@ boolean isDefaultOption(String possibleValuesJson) {
          return isDefault;
     }
 
+    boolean isElasticExportWorkflowType(String action) {
+        if (StringUtils.isBlank(action)) return false;
+        List<Map<String, String>> actions = new Gson().fromJson(action, List.class);
```

Comment:
Use Value[] instead of List<Map<String, String>> because actions can have type, type2 and so one and not be key and value pair